tools/golangci-lint/Makefile

```diff
@@ -0,0 +1,2 @@
+golangci-lint:
+	go build -o golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
do we need these golangci stuff in this PR ?

Reply:
argh no we do not. I accidentally added it